zipkin/src/main/java/zipkin2/internal/JsonCodec.java

```diff
@@ -222,7 +221,9 @@ public static <T> void writeList(WriteBuffer.Writer<T> writer, List<T> value, Wr
 
   static IllegalArgumentException exceptionReading(String type, Exception e) {
     String cause = e.getMessage() == null ? "Error" : e.getMessage();
-    if (cause.contains("Expected BEGIN_OBJECT") || cause.contains("malformed")) {
+    if (cause.contains("Expected BEGIN_OBJECT")
```

Comment:
while this normalizes the error as intended, this PR already shows any data that has malformed tags in zipkin were created out-of-band, like not using zipkin, as all collectors regardless of HTTP or Kafka use this path